apps/nfw-react/src/app/project/cmp/Cart/_cart.scss

```diff
@@ -0,0 +1,129 @@
+.cart {
+  display: grid;
+  grid-template-columns: 30px auto 30px;
+  grid-template-rows: 10vh 6rem 60vh;
+
+  grid-template-areas:
+    '. . .'
+    '. cart_heading .'
+    '. cart_container .';
+
+  &__heading {
+    grid-area: cart_heading;
+    justify-self: center;
+    align-self: center;
+    font-size: 2.5rem;
+    margin-bottom: 0.5rem;
+    color: $secondary-color;
+    transition: 0.5s ease-out;
+
+    &--one {
```

Comment:
(should) `.cart__heading--one` ? `one` isn't modifier but element so ` .cart__heading-one` ;)